Makefile.am

```diff
@@ -439,28 +436,33 @@ libplugin_mock_la_SOURCES = \
 	src/daemon/utils_time.h
 
 libplugin_mock_la_CPPFLAGS = $(AM_CPPFLAGS) -DMOCK_TIME
-libplugin_mock_la_LIBADD = libcommon.la libignorelist.la $(COMMON_LIBS)
+libplugin_mock_la_LIBADD = libcommon.la libignorelist.la libmetadata.la $(COMMON_LIBS)
+#libplugin_mock_la_LIBADD = libcommon.la libignorelist.la $(COMMON_LIBS)
```

Comment:
Can we get rid of this line? Looks like an oversight.

Reply:
Done.